test/e2e/cp_test.go

```diff
@@ -91,6 +91,49 @@ var _ = Describe("Podman cp", func() {
 		Expect(roundtripContent).To(Equal(originalContent))
 	})
 
+	// Copy a file to the container, then back to the host in --pid=host
+	It("podman cp --pid=host file", func() {
```

Comment:
To make ubuntu happy.

```suggestion
	It("podman cp --pid=host file", func() {
	SkipIfRootlessCgroupsV1("Not supported for rootless + CGroupsV1")
```